client/src/components/layout/Sidebar.js

```diff
@@ -1,16 +1,10 @@
 import "./Sidebar.css"
-import {SidebarData} from './SidebarData'
-import { Link, useLocation } from 'react-router-dom'
```

Comment:
I think Sidebar.js is using Link but you've removed it. Can you check the error messages from Static Analyzer? And also the `Unexpected use of 'location'` from Static Analyzer

Reply:
Yep, this is removed to make the Sidebar.js as a reusabale component should not have this in the sidebar component anymore.